fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/LoanApplicationTerms.java

```diff
@@ -69,6 +69,7 @@ public final class LoanApplicationTerms {
     private final Integer repaymentEvery;
     private final PeriodFrequencyType repaymentPeriodFrequencyType;
 
+    private long variationDays = 0L;
```

Comment:
Can you please explain this logic?

Reply:
For Reschedule we have variations and each variation usually has a change in a date, so we have the original due date and the new or reschedule date, then in this variable we can store the difference in days to be consider in the case of Fixed Length